### PR TITLE
Split shader tab

### PR DESCRIPTION
**Pull request by johnplate**
This request cleaned up the shader tab from features that used to be required for OpenGL support, and splits it into a list and a view tab, similar to the texture tab.

Fourth (and last) PR for b/170298289

### CODE REVIEW

File: gapic/src/main/com/google/gapid/views/ShaderView.java [hunk 1]
```diff
@@ -218,519 +125,106 @@ public Control getControl() {
...
-    if (models.commands.isLoaded() && models.resources.isLoaded()) {
-      if (models.commands.getSelectedCommands() == null) {
-        loading.showMessage(Info, Messages.SELECT_COMMAND);
-      } else {
-        loading.stopLoading();
-      }
+  @Override
+  public void onResourcesLoaded() {
+    if (!pinned) {
+      loading.showMessage(Info, Messages.SELECT_TEXTURE);
```
Comment (pmuetschard) [inline gapic/src/main/com/google/gapid/views/ShaderView.java:169]: This is the wrong message.

File: gapic/src/main/com/google/gapid/views/ShaderView.java [hunk 2]
```diff
@@ -218,519 +125,106 @@ public Control getControl() {
...
+      onCaptureLoadingStart(false);
+    } else {
+      loadShader(models.resources.getSelectedShader());
+    }
   }
 
   @Override
-  public void onCaptureLoadingStart(boolean maintainState) {
-    loading.showMessage(Info, Messages.LOADING_CAPTURE);
+  public boolean isPinnable() {
```
Comment (pmuetschard) [inline gapic/src/main/com/google/gapid/views/ShaderView.java:136]: Missing `supportsPinning()` returning true.

**Review by pmuetschard — COMMENTED** [review]
Please also rebase on top of master, since there's merge conflicts (I moved `API.ResourceType` to `Path.ResourceType` and renamed the enum constants).

File: gapic/src/main/com/google/gapid/views/ShaderView.java (hunk 1 above)
Comment (johnplate) [reply gapic/src/main/com/google/gapid/views/ShaderView.java]: Oops, thanks, fixed

File: gapic/src/main/com/google/gapid/views/ShaderView.java (hunk 2 above)
Comment (johnplate) [reply gapic/src/main/com/google/gapid/views/ShaderView.java:141]: Thanks, good catch. Fixed

**Review by pmuetschard — APPROVED** [review]

**Pull request merged**